tests/particle_test.cc

```diff
@@ -145,10 +145,23 @@ TEST_CASE("Particle is checked for 1D case", "[particle][1D]") {
         std::make_shared<mpm::Particle<Dim>>(id, coords);
 
     // Check mass
-    REQUIRE(particle->mass() == Approx(0.0).epsilon(Tolerance));
-    double mass = 100.5;
+    REQUIRE(particle->mass() == Approx(0.).epsilon(Tolerance));
+    double mass = 201.0;
```

Comment:
Any thoughts why change the value of the test?

Reply:
That's just to check the return and assign values to mass, no different. I will revert to minimize the change. Thanks for pointing that out.